Lib/zipapp.py

```diff
@@ -177,6 +179,49 @@ def get_interpreter(archive):
         if f.read(2) == b'#!':
             return f.readline().strip().decode(shebang_encoding)
 
+def _normalize_patterns(values: Iterable[str] | None) -> list[str]:
+    """
+    Return patterns exactly as provided by the CLI (no comma splitting).
+    Each item is stripped of surrounding whitespace; empty items are dropped.
+    """
+    if not values:
+        return []
+    out: list[str] = []
+    for v in values:
+        v = v.strip()
+        if v:
+            out.append(v)
+    return out
```

Comment:
This function doesn't feel very useful. Why not just inline its logic into `_make_glob_filter`?

Reply:
I have made it inline